source/install/install-docker.rst

```diff
@@ -150,16 +129,37 @@ To deploy Mattermost on Docker:
       
 7. Create your first Mattermost system admin user, :doc:`invite more users </collaborate/manage-channel-members>`, and explore the Mattermost platform. 
 
+Configure SSO With GitLab (Optional)
```

Comment:
Is this normal for the docker builds? Wondering if this needs to be moved to a new SSO page

Reply:
It was already there, so I left it. 0/5 on moving it out of this doc, but these instructions are Docker specific

Reply:
Given that it's Docker-specific, I'm inclined to leave here and reconsider as part of the restructuring work in progress.